server/core/src/irods_logger.cpp

```diff
@@ -126,7 +129,7 @@ void log::init(bool _write_to_stdout, bool _enable_test_mode) noexcept
     }
     else {
         static const char* id = "";
-        sinks.push_back(std::make_shared<spdlog::sinks::syslog_sink>(id, LOG_PID, LOG_LOCAL0));
+        sinks.push_back(std::make_shared<spdlog::sinks::syslog_sink_st>(id, LOG_PID, LOG_LOCAL0, false));
```

Comment:
@korydraughn - Has the `spdlog::sinks::syslog_sink` been used in a multi-threaded context this whole time, or single-threaded? I assumed single-threaded because we are using `spdlog::details::null_mutex` for the other two sinks, but let me know if I misread this.

Reply:
i actually have these same changes along with others to make this work (due to logical quotas).  there are changes that need to be made to force spdlog to use our packaged fmt library as well.  currently, spdlog will choose to use it's internal version of fmt.

Reply:
Got it, thanks. I'll close this PR, but I'll keep the PR just in case.